cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -143,7 +143,7 @@ runNode nodeCli@NodeCLIArguments{..} loggingLayer cc = do
         SomeProtocol p  <- fromProtocol cc protocol
         let tracers     = mkTracers traceOptions trace'
         case viewMode of
-          SimpleView -> handleSimpleNode p nodeCli myNodeAddress topology trace' tracers cc
+          SimpleView -> handleSimpleNode p nodeCli myNodeAddress topology trace' tracers (cc { ccDBPath = dbFp })
```

Comment:
Insertion of the db filepath into `CardanoConfiguration`

Reply:
Hmm, I'm not sure it's the best place for the configuration composition.  But it's hard to tell, with all the code movement.

Reply:
Ideally, the `cc` -- `CardanoConfiguration` -- should have the CLI args transparently `<>`'ed in the Partial Options Monoid style, in a single place -- so that we don't have to do one-off splicings like here.